src/locales/strings/enUS.json

```diff
@@ -79,7 +79,12 @@
   "fragment_send_send_bitcoin_unscannable": "Unable to scan QR code",
   "scan_invalid_address_error_title": "Invalid Address",
   "scan_invalid_address_error_description": "Not a valid public address",
+  "fragment_send_fio_address_dialog_title": "Request by FIO Address",
   "fragment_send_send_to_hint": "Public Address or Private Key",
+  "fragment_send_send_to_fio_error": "Invalid FIO Address",
```

Comment:
Remove "fragment_" from newly added key names. Do not change any existing keys. This is legacy naming.

Reply:
changed